node/src/BaseClient.ts

```diff
@@ -571,6 +574,17 @@ export interface BaseClientConfiguration {
      * used.
      */
     inflightRequestsLimit?: number;
+    /**
+     * Availability Zone of the client.
+     * This setting ensures that read operations are directed to nodes within the specified AZ.
+     * If not set, the client's AZ will be determined dynamically based on the node addresses or default behavior.
```

Comment:
What does it mean base on node address? can you elaborate?

Reply:
Actually thats not the case.
The client does not have built-in logic to determine its AZ (afaik) 
If it's not specified, it would just operate without az awarness

Reply:
So lets say it as it is.
At the end of the day our users can see, and some actually check our code. We can't say we have a feature that we don't.

Reply:
@adarovadya  if you could approve ^